core/src/main/scala/kafka/tools/ConsoleConsumer.scala

```diff
@@ -315,11 +321,18 @@ object ConsoleConsumer extends Logging {
 
     formatter.configure(formatterArgs.asScala.asJava)
 
-    val topicOrFilterOpt = List(topicOpt, whitelistOpt).filter(options.has)
-    if (topicOrFilterOpt.size != 1)
-      CommandLineUtils.printUsageAndDie(parser, "Exactly one of whitelist/topic is required.")
     topicArg = options.valueOf(topicOpt)
-    whitelistArg = options.valueOf(whitelistOpt)
+    includedTopicsArg = if (options.has(includeOpt))
+      options.valueOf(includeOpt)
+    else
+      options.valueOf(whitelistOpt)
+
+    val noTopicOrFilterSpecified = topicArg == null && includedTopicsArg == null
+    val bothTopicAndFilterSpecified = topicArg != null && includedTopicsArg != null
+
+    if (noTopicOrFilterSpecified || bothTopicAndFilterSpecified)
```

Comment:
you could also write this as an XOR `(topicArg != null ^ includedTopicsArg != null)`, and add a comment, not sure which one is more readable.

Reply:
I revert it back to 
```
val topicOrFilterArgs = List(topicArg, includedTopicsArg).filterNot(_ == null)
if (topicOrFilterArgs.size != 1) {
//handle error
}
```
And added a comment as well is seems more readable than the XOR and my previous handling

Reply:
I don't have a strong preference, feel free to use whichever approach seems more readable to you